openfe/tests/conftest.py

```diff
@@ -373,3 +379,141 @@ def am1bcc_ref_charges():
     HAS_ESPALOMA = True
 except ModuleNotFoundError:
     HAS_ESPALOMA = False
+
+
+@pytest.fixture(scope="module")
+def chlorobenzene():
+    """Load chlorobenzene with partial charges from sdf file."""
+    with resources.files("openfe.tests.data.htf") as f:
+        return SmallMoleculeComponent.from_sdf_file(f / "t4_lysozyme_data" / "chlorobenzene.sdf")
+
+
+@pytest.fixture(scope="module")
+def fluorobenzene():
+    """Load fluorobenzene with partial charges from sdf file."""
+    with resources.files("openfe.tests.data.htf") as f:
+        return SmallMoleculeComponent.from_sdf_file(f / "t4_lysozyme_data" / "fluorobenzene.sdf")
+
+
+@pytest.fixture(scope="module")
+def chlorobenzene_to_fluorobenzene_mapping(chlorobenzene, fluorobenzene):
+    """Return a mapping from chlorobenzene to fluorobenzene."""
+    return LigandAtomMapping(
+        componentA=chlorobenzene,
+        componentB=fluorobenzene,
+        componentA_to_componentB={
+            # perfect one-to-one mapping
+            0: 0,
+            1: 1,
+            2: 2,
+            3: 3,
+            4: 4,
+            5: 5,
+            6: 6,
+            7: 7,
+            8: 8,
+            9: 9,
+            10: 10,
+            11: 11,
+        },
+    )
+
+
+@pytest.fixture(scope="module")
+def t4_lysozyme_solvated():
+    """Load the T4 lysozyme L99A structure and solvent from the pdb file."""
+    with resources.files("openfe.tests.data.htf") as f:
+        return ProteinComponent.from_pdb_file(
+            (f / "t4_lysozyme_data" / "t4_lysozyme_solvated.pdb").as_posix()
```

Comment:
This is failling in py3.13, that being said, the gzip thing might fix it anyways.